README.md

```diff
@@ -18,3 +18,397 @@ We are looking for more hands to join in - or review the task list so that we ca
 See the (draft) [documentation](https://ascii-supply-networks.github.io/dagster-slurm/)
 
 > We are actively looking for contributions to bring this package to life together
+
+
+## developing
+
+```bash
+docker compose up --build -d
```

Comment:
The `docker login` error message could be avoided if the command here was broken up into
```
docker compose build
docker compose up -d
```
I initially found the `docker login` error messages confusing... the `docker compose up --build -d` command does succeed, but it was unclear at the start whether it would when I first saw the error messages. But those probably only occur when you haven't built these locally before.

Reply:
documented as 2 steps for now.

Reply:
Yeah I totally understand the desire to keep the toolchain simple for now!